CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
   Parameter defaults can be set in the task metadata file and will be used if no value is supplied for the parameter.
 
+* **New `remove_from_group` plan function** ([#1418](https://github.com/puppetlabs/bolt/issues/1418))
+
+  The new plan function, `remove_from_group`, allows you to remove a target from an inventory group during plan execution.
+
```

Comment:
Should we comment on any hickups this may present? This copy is nice and straightforward, but if there are any unforeseen issues we might want to comment on what they are and how to avoid them?